integration-cli/docker_api_swarm_node_test.go

```diff
@@ -40,17 +43,17 @@ func (s *DockerSwarmSuite) TestAPISwarmNodeUpdate(c *check.C) {
 	})
 
 	n := d.GetNode(c, nodes[0].ID)
-	c.Assert(n.Spec.Availability, checker.Equals, swarm.NodeAvailabilityPause)
+	assert.Equal(c, n.Spec.Availability, swarm.NodeAvailabilityPause)
 }
 
-func (s *DockerSwarmSuite) TestAPISwarmNodeRemove(c *check.C) {
+func (s *DockerSwarmSuite) TestAPISwarmNodeRemove(c *testing.T) {
 	testRequires(c, Network)
 	d1 := s.AddDaemon(c, true, true)
 	d2 := s.AddDaemon(c, true, false)
 	_ = s.AddDaemon(c, true, false)
 
 	nodes := d1.ListNodes(c)
-	c.Assert(len(nodes), checker.Equals, 3, check.Commentf("nodes: %#v", nodes))
+	assert.Equal(c, len(nodes), 3, fmt.Sprintf("nodes: %#v", nodes))
```

Comment:
the `fmt.Sprintf(..)` is not needed, `assert.Equal()` already takes a format-string; https://github.com/moby/moby/blob/c147e9e1a45b1c66bc5773814b53bcd27132557c/vendor/gotest.tools/assert/assert.go#L256



```go
func TestFoo(t *testing.T) {
	assert.Equal(t, "one", "two", "bla %s bla", "bla")
}
```

```
=== RUN   TestFoo
--- FAIL: TestFoo (0.00s)
    debug_test.go:48: assertion failed: one (string) != two (string): bla bla bla
FAIL

Process finished with exit code 1

Reply:
It could probably remove all `check.Commentf(...)`, but we can do in a follow-up